docs/source/cli/model-training-inference/output.rst

```diff
@@ -0,0 +1,169 @@
+.. _gs-output:
+
+GraphStorm Output
+=================
+
+.. _gs-output-embs:
+
+Saved Node Embeddings
+---------------------
+When ``save_embed_path`` is provided in the training config or inference condig,
+GraphStorm will save the node embeddings in the corresponding path. The node embeddings
+of each node type are saved separately under different sub-directories named with
+the corresponding node types. GraphStorm will also save an ``emb_info.json`` file,
+which contains all the metadata for the saved node embeddings. The ``save_embed_path``
+will look like following:
+
+.. code-block:: bash
+
+    emb_dir/
+        ntype0/
+            embed_nids-00000.pt
+            embed_nids-00001.pt
+            ...
+            embed-00000.pt
+            embed-00001.pt
+            ...
+        ntype1/
+            embed_nids-00000.pt
+            embed_nids-00001.pt
+            ...
+            embed-00000.pt
+            embed-00001.pt
+            ...
+        ...
+        emb_info.json
+
+The ``embed_nids-*`` files store the integer node IDs of each node embedding and
```

Comment:
If the file name and file format change, need to revise these two sentences two reflect these changes.

Reply:
Sure. But I don't think we will change it in the future.